src/components/feed/CreateFeed/createfeed.scss

```diff
@@ -263,3 +264,10 @@ Styles for pipelines
     }
   }
 }
+
+/*calender fix */
+.pf-c-input-group input:not([type=checkbox]):not([type=radio]), .pf-c-input-group textarea {
```

Comment:
Please work on a branch from within your fork. You should be creating a branch per feature (per issue, per PR, ...)